src/api/commonTypes.ts

```diff
@@ -92,3 +93,9 @@ export interface ICurrencyOptions {
   format: CurrencyFormat;
   symbol?: string;
 }
+
+export interface CommentMention {
+  position: number;
+  length: number;
+  user: MentionedUser;
+}
```

Comment:
I suggest declaring this interface in '/src/api/User/types' (or moving `MentionedUser` and the related interfaces here), so that we avoid having a dependency cycle.